app/models/coronavirus/live_stream.rb

```diff
@@ -1,3 +1,3 @@
 class Coronavirus::LiveStream < ApplicationRecord
-  validates :url, url: true, presence: true
+  validates :url, youtube_url: true, presence: true
```

Comment:
What's the difference between `url` and `youtube_url`? Or does this make `youtube_url` an alias of `url`?

Reply:
It represents the renaming of the underlying validator.

Prior to this change we had a validator, UrlValidator, which had the job of validating that a string was a YouTube URL. This was rather surprising as the name, UrlValidator, implies that it validates that a string is a URL. I renamed that to be YoutubeUrlValidator to reflect that it actually validates that a string is a YouTube URL.